backend/app/tests/api/routes/test_prompts.py

```
@@ -0,0 +1,277 @@
+    client: TestClient,
+    db: Session,
+    user_api_key: APIKeyPublic,
+):
+    """Test retrieving prompts with specific skip and limit values"""
+    project_id = user_api_key.project_id
+
+    for i in range(3):
+        create_test_prompt(db, project_id, name=f"prompt_{i}")
+
+    response = client.get(
+        f"/api/v1/prompts/?skip=1&limit=1",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)